ftw/solr/commands.py

```diff
@@ -23,7 +24,6 @@ def setup_site(app, options):
     if not portal:
         sys.exit("Plone site not found at %s" % options.plone_site)
 
-    user = portal.getOwner()
```

Comment:
you may also need to acquisition wrap the special user as in https://github.com/4teamwork/opengever.core/blob/04fcdf52cae62e3ccacfa6c0ff3b3d5a4d9c9ebf/opengever/core/debughelpers.py#L46

things may or may not break depending on what we do with the user.